ext/drbbdup/drbbdup.c

```diff
@@ -1349,7 +1430,9 @@ drbbdup_instrument_dups(void *drcontext, void *tag, instrlist_t *bb, instr_t *in
         if (is_last_special) {
             pt->last_instr = last;
         } else {
-            instr_t *prev = instr_get_prev(end_instr);
+            instr_t *prev = instr_get_prev(end_initial);
+            if (drbbdup_is_our_emulation_marker(prev))
```

Comment:
I don't get this branch. From my reading `end_initial` should be the start emu label, and we called instr_get_prev() to get the prior instr in the line before. Why do we have this branch. May you add a comment explaining it.

Reply:
Hmm...looks like this was left from a prior version where there was no separate `end_initial`.  Unfortunately I'm getting sloppy here it seems, trying to get this all tied up too quickly to get the #3995 going.  Removing this if().